docs/en/entity-onboarding.rst

```diff
@@ -0,0 +1,1095 @@
+"""""""""""""""""""""
+
+Federation participants validate Trust Mark status through two mechanisms:
+
+1. **Static Validation**: Cryptographic verification using the issuing Federation Authority's public key from the trust chain.
+2. **Dynamic Validation**: Real-time status verification via the issuing Federation Authority's ``/trust_mark_status`` endpoint as defined in :ref:`trust:Federation API endpoints`.
+
+
+Certificate Management Operations
+----------------------------------
+
+This section defines the operational procedures for X.509 certificate management within the IT-Wallet federation, covering certificate chain analysis, validation procedures, and revocation verification. These procedures complement the federation onboarding processes and support ongoing certificate lifecycle management for all federation participants.
+
+Federation PKI Architecture
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The IT-Wallet federation operates a hierarchical Public Key Infrastructure where:
+
+	- **Trust Anchor**: Acts as Root Certificate Authority where root certificates MUST NOT exceed **5-year validity period**.
+	- **Federation Entity Certificate**: Each federation participant receives a certificate that operates as a limited sub-CA where certificates MUST NOT exceed **2-year validity period**.
+	- **Protocol Certificates**: Self-issued certificates for internal services where certificates SHOULD NOT exceed **1-year validity period**.
+
+Each federation entity MUST expose its Federation Entity certificate on a publicly accessible endpoint. The Federation Entity private key serves dual purposes:
+
+	1. Self-issuing Protocol certificates for internal cryptographic operations (limited sub-CA capability).
+	2. Acting as the Federation Entity Key for signing Entity Statements.
+
+.. note:: 
+  Federation entities (Leafs) can ONLY issue certificates for themselves (Protocol certificates), NOT for other federation entities. Only Federation Authorities (Trust Anchor and Intermediates) can issue certificates for other entities.
+
+For Protocol certificates with validity periods exceeding 24 hours, the issuing entity MUST publish and regularly update a Certificate Revocation List (CRL) on a publicly accessible endpoint.
+
+Certificate Chain Structure and Analysis
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Federation entities receive certificate chains during the onboarding process. Understanding and validating these chains is essential for proper federation operations and trust verification.
+
+Certificate Chain Visualization
+""""""""""""""""""""""""""""""""
+
+Federation entities SHOULD analyze received certificate chains using standard cryptographic tools to verify proper structure and validate trust relationships.
+
+The following script enables federation entities to:
+
+	- Extract certificate details for verification.
+	- Analyze certificate extensions and constraints.
+	- Verify certificate hierarchy and relationships
+
+.. code-block:: bash
```

Comment:
please move into the folder utils/tools (we have) and give a href to the git file